nettrace/httpclient.go

```diff
@@ -323,6 +330,8 @@ func (c *HTTPClient) publishTrace(t networkTrace) {
 // resetTraces : recreates all maps holding recorded network traces and pcaps.
```

Comment:
Please remove from the function comment: `The function should be called with HTTPClient locked.`

Reply:
Good catch, done.

There were a lot of intertwined private/public, locking/not-locking funcs here; I had lots of testing that just hung and eventually timed out, but eventually figured it all out.